src/core/interaction/trackball.cpp

```diff
@@ -112,6 +114,9 @@ Trackball::Trackball(TrackballObject* object)
     addProperty(sensitivity_);
     addProperty(movementSpeed_);
     addProperty(fixUp_);
+    addProperty(worldUp_);
+    worldUp_.setSelectedIndex(1);
```

Comment:
default selected index can be set as the 4th argument in the constructor 
https://github.com/inviwo/inviwo/blob/23770f502f7a2596eb34f43286f21216fdd57750/include/inviwo/core/properties/optionproperty.h#L121-L125